File: 4-median-of-two-sorted-arrays/4-median-of-two-sorted-arrays.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        merged_arr = nums1 + nums2
        merged_arr.sort()
        
        if len(merged_arr) % 2 == 0:
            return (merged_arr[int(len(merged_arr) / 2) - 1] + merged_arr[int(len(merged_arr) / 2)]) / 2
        else:
            return (merged_arr[len(merged_arr) // 2])
        

File: 4-median-of-two-sorted-arrays/4-median-of-two-sorted-arrays.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number}
 */
var findMedianSortedArrays = function(nums1, nums2) {
    let merged_arr = nums1.concat(nums2);
    merged_arr.sort();
    
    merged_arr.sort(function(a, b) {
        return a - b;
    });
    
    if(merged_arr.length % 2 == 0){
        let left = merged_arr[parseInt(merged_arr.length / 2 - 1)]; 
        let right = merged_arr[parseInt(merged_arr.length / 2)];
        return (left + right) / 2;
    }
    else{
        return merged_arr[parseInt(merged_arr.length / 2)];
    };
};